src/main/webapp/app/modules/login/login-form.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
-import { TextField, Flex, Form, Checkbox, Button, Text } from '@adobe/react-spectrum';
+import { TextField, Flex, Checkbox, Button, Text } from '@adobe/react-spectrum';
 import Typography from '@material-ui/core/Typography';
```

Comment:
Do not use material typography please use react spectrum <Text> and spectrum css classes. Refer to header component/avatar

Reply:
i do not see class used in header component..can you mention here which class and tag was used?
pls note that bold

also import Typography from '@material-ui/core/Typography'; was used in home component as well

Reply:
Its present in data-studio-avatar.tsx file. Please refer this documentation
https://opensource.adobe.com/spectrum-css/typography-body.html

Just realised that the version is upgraded to 3.0.x-beta. We will updated this in a new PR

Regarding the home component we will remove it  and thanks for finding.